docs/site/OpenAPI-generator.md

```diff
@@ -0,0 +1,245 @@
+}
+```
+
+src/models/id-type.model.ts:
+
+```ts
+export type IdType = string | number;
+```
+
+src/models/pet.model.ts:
+
+```ts
+import {NewPet} from './new-pet.model.ts';
+export type Pet = NewPet & {id: number};
+```
+
+2.  The generator groups operations (`paths.<path>.<verb>`) by tags. If no tag
+    is present, it defaults to `OpenApi`. For each tag, a controller class is
+    generated as `src/controllers/<tag-name>.controller.ts` to hold all
+    operations with the same tag.
+
+For example,
+
+```ts
+import {operation, param} from '@loopback/rest';
+import {DateTime} from '../models/date-time.model';
+
+/**
+ * The controller class is generated from OpenAPI spec with operations tagged
+ * by account
+ *
+ */
+export class AccountController {
+  constructor() {}
+
+  /**
+   * Get list of carts.
+   */
+  @operation('get', '/account.cart.list.json')
+  async accountCartList(
+    @param({name: 'params', in: 'query'})
+    params: string,
+    @param({name: 'exclude', in: 'query'})
+    exclude: string,
+    @param({name: 'request_from_date', in: 'query'})
+    request_from_date: string,
+    @param({name: 'request_to_date', in: 'query'})
+    request_to_date: string,
+  ): Promise<{
+    result?: {
+      carts?: {
+        cart_id?: string;
+        id?: string;
+        store_key?: string;
+        total_calls?: string;
+        url?: string;
+      }[];
+      carts_count?: number;
+    };
+    return_code?: number;
+    return_message?: string;
+  }> {
+    throw new Error('Not implemented');
+  }
+
+  /**
+   * Update configs in the API2Cart database.
+   */
+  @operation('put', '/account.config.update.json')
+  async accountConfigUpdate(
+    @param({name: 'db_tables_prefix', in: 'query'})
+    db_tables_prefix: string,
+    @param({name: 'client_id', in: 'query'})
+    client_id: string,
+    @param({name: 'bridge_url', in: 'query'})
+    bridge_url: string,
+    @param({name: 'store_root', in: 'query'})
+    store_root: string,
+    @param({name: 'shared_secret', in: 'query'})
+    shared_secret: string,
+  ): Promise<{
+    result?: {
+      updated_items?: number;
+    };
+    return_code?: number;
+    return_message?: string;
+  }> {
+    throw new Error('Not implemented');
+  }
+
+  /**
+   * List webhooks that was not delivered to the callback.
+   */
+  @operation('get', '/account.failed_webhooks.json')
+  async accountFailedWebhooks(
+    @param({name: 'count', in: 'query'})
+    count: number,
+    @param({name: 'start', in: 'query'})
+    start: number,
+    @param({name: 'ids', in: 'query'})
+    ids: string,
+  ): Promise<{
+    result?: {
+      all_failed_webhook?: string;
+      webhook?: {
+        entity_id?: string;
+        time?: DateTime;
+        webhook_id?: number;
+      }[];
+    };
+    return_code?: number;
+    return_message?: string;
+  }> {
+    throw new Error('Not implemented');
+  }
+}
+```
+
+### OpenAPI Examples
+
+Try out the following specs with `lb4 openapi`:
```

Comment:
My first thought was that this command is limited to the following specs only. Might be better to say `try out the following specs or use your own with `lb4 openapi`.